lib/senkyoshi/models/file.rb

```diff
@@ -32,8 +32,13 @@ def extract_file(entry)
       name = "#{@dir}/#{entry.name}"
       path = File.dirname(name)
       FileUtils.mkdir_p path unless Dir.exist? path
-      entry.extract(name)
-      name
+
+      begin
```

Comment:
I'm not a fan of open ended rescues like this. Something unrelated could fail and it will just get swallowed up in the rescue. Is there another way you can check for failures?